### ISSUE TITLE
Node engine version

### ISSUE THREAD

**Issue by olivercoad** [demtario/hes-gallery#15]
When I try to install, I get an error due to not compatible node version.

```
$ npm i -D hes-gallery
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: hes-gallery@1.5.0
npm ERR! notsup Not compatible with your version of node/npm: hes-gallery@1.5.0
npm ERR! notsup Required: {"node":"11.x"}
npm ERR! notsup Actual:   {"npm":"8.1.4","node":"v17.2.0"}
```

**Comment by demtario** [comment]
I'll try to resolve this in the following week. Probably engine requirement is not needed at all

**Comment by olivercoad** [comment]
Yes, I think you can just remove the engine requirement altogether.

I ended up just setting `engine-strict=false` in `.npmrc` which allowed npm to install it anyway (warning instead of error).

**Comment by demtario** [comment]
I removed engine requirement and it should work now.

Fixed in version `1.5.1`
